source/features/copy-on-y.tsx

```diff
@@ -1,29 +1,20 @@
-import select from 'select-dom';
-import * as pageDetect from 'github-url-detection';
-
 import features from '../feature-manager';
 import {isEditable} from '../helpers/dom-utils';
 
-const handler = ({key, target}: KeyboardEvent): void => {
+async function handler({key, target}: KeyboardEvent): Promise<void> {
 	if (key === 'y' && !isEditable(target)) {
-		const permalink = select('a.js-permalink-shortcut')!.href;
-		void navigator.clipboard.writeText(permalink + location.hash);
+		const url = location.href;
+		await navigator.clipboard.writeText(url);
+		// Log to ensure we're coping the new URL
+		console.log('Copied URL to the clipboard', url);
```

Comment:
Your keeping this here?

Reply:
Yes for now:

- could be helpful if suddenly this starts copying the non-permalinked URL
- we're not giving any feedback that this feature is doing anything
- the log only appears in rare situations, i.e. when the user does tap Y

Reply:
Why not use the logger function?

Reply:
Laziness to check how that worked 😂 

Will do 🚀

Reply:
We don't seem to use that anywhere except "show features that are run on the page", it doesn't seem strictly related to this type of logging. Given this is user feedback, I'd still keep it here unconditionally